exts/betterTooltip/betterTooltip.js

```diff
@@ -64,6 +160,20 @@
                 return _categoryLabelFormatString;
             };
 
+            /**
+             * Gets or sets the format function for the label of the category.
+             *
+             * The default format function uses the categoryLabelFormatString property for creating the label.
+             *
+             * Custom formaters receive the full tooltip model object, the category being formated and the
```

Comment:
Custom formatters receive the full tooltip model object, the category being formatted and the